test/table_test.go

```diff
@@ -519,6 +519,16 @@ func TestTable(t *testing.T) {
 			}
 		})
 	})
+	t.Run("SessionsShutdown", func(t *testing.T) {
+		time.Sleep(10 * time.Second)
```

Comment:
sleep is best avoided otherwise the tests may flaps.

Reply:
ack